win32/CMakeLists.txt

```diff
@@ -236,36 +236,66 @@ if(USE_SE_SM)
 
 endif()
 
-if (CMAKE_BUILD_TYPE EQUAL "DEBUG")
-set(GLSLANG_SUFFIX "d")
-endif()
+add_library(glslangd STATIC IMPORTED GLOBAL)
+set_target_properties(glslangd PROPERTIES
+  IMPORTED_LOCATION ${CMAKE_CURRENT_LIST_DIR}/libs/glslang/glslangd.lib
```

Comment:
https://cmake.org/cmake/help/v3.0/prop_tgt/IMPORTED_LOCATION_CONFIG.html is helpful
```
set_target_properties(glslangd PROPERTIES
IMPORTED_LOCATION_DEBUG ${CMAKE_CURRENT_LIST_DIR}/libs/glslang/glslangd.lib
IMPORTED_LOCATION_RELEASE ${CMAKE_CURRENT_LIST_DIR}/libs/glslang/glslang.lib
endif()	  
```